libs/internal/console-driver/test-util/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@ngworker/internal/console-driver/test-util",
```

Comment:
I had problems with the application because now Nx loads the config files automatically and it was having problems identifying what was the correct config file, the project.json or the package.json. 

Does specifying the name like that fix it?

Reply:
No, this is only needed to make this library buildable.